doc/rvv-intrinsic-spec.adoc

```diff
@@ -1,24 +1,24 @@
 == Introduction
 
-The RISC-V vector C intrinsics provides users interfaces in the C language level to directly leverage the RISC-V "V" extension ^0^ (also abbreviated as "RVV"), with help from the compiler to handle instruction scheduling and register allocation.
+The RISC-V vector C intrinsics provides users interfaces in the C language level to directly leverage the RISC-V "V" extension ^0^ (also abbreviated as "RVV"), with help from the compiler to handle instruction scheduling and register allocation. The RVV C intrinsics also aims to free users from responsibility to maintain the correct configuration setting ^18^ for the vector instruction executions.
 
-This document will use the term "v-spec" to indicate the RISC-V "V" extension specification ^0^.
+This document uses the term "v-spec" to indicate the RISC-V "V" extension specification ^0^. This document also uses the term "RVV" as an abbreviation for the RISC-V "V" extension.
```

Comment:
> This document uses the term "v-spec" to indicate the RISC-V "V" extension specification ^0^

We wouldn't say "f-spec" to refer to the F-extension specification. These are ratified standard extensions of the RISC-V ISA. I think we should just say "the ISA spec" (or "specification").

Many standard extensions are developed in repositories separate from https://github.com/riscv/riscv-isa-manual , but after ratification they are merged into this document. Here is the PR to merge the (ratified) vector extension: https://github.com/riscv/riscv-isa-manual/pull/1088.

Until this is merged, it is OK to cite https://github.com/riscv/riscv-v-spec. Or perhaps cite both, with a comment in the bibliography.